<map id="django.contrib.gis.utils.layermapping.InvalidInteger" name="django.contrib.gis.utils.layermapping.InvalidInteger">
<area shape="rect" id="Node000001" title=" " alt="" coords="5,167,273,207"/>
<area shape="rect" id="Node000002" href="$classdjango_1_1contrib_1_1gis_1_1utils_1_1layermapping_1_1LayerMapError.html" title=" " alt="" coords="5,79,273,119"/>
<area shape="poly" id="edge1_Node000001_Node000002" title=" " alt="" coords="142,133,142,166,137,166,137,133"/>
<area shape="rect" id="Node000003" title=" " alt="" coords="96,5,183,31"/>
<area shape="poly" id="edge2_Node000002_Node000003" title=" " alt="" coords="142,44,142,79,137,79,137,44"/>
</map>
